test/exports.spec.js

```diff
@@ -1,32 +1,31 @@
-import * as chai from 'chai'
-import * as path from 'path';
-import * as glob from 'glob';
-import toCamelCase from 'camelcase';
-import interopRequire from 'interop-require';
-import * as decorators from '../';
+const chai = require('chai');
+const path = require('path');
+const glob = require('glob');
+const camelCase = require('camelCase');
```

Comment:
This is what is causing the error on Linux. This import string needs to be `camelcase` all lowercase. Linux filesystem is case-sensitive.